tests/search/search-page.spec.ts

```diff
@@ -47,7 +49,7 @@ test.describe('Basic Search tests', () => {
     });
 
     await test.step(`Searching and search result count should be displayed`, async () => {
-      await searchPage.collectionFacets.checkResultCount();
+      await searchPage.collectionFacets.displaysResultCount();
```

Comment:
question: can we group those awaits into 1 test.step (the last one)?

```suggestion
      await searchPage.clickSearchInputOption(SearchOption.TEXT);
      await searchPage.queryFor('dogs');
      await searchPage.collectionFacets.displaysResultCount();
```

Reply:
added test steps for better tests summary for the test report: https://timdeschryver.dev/blog/keep-your-playwright-tests-structured-with-steps#code-comments